integration_tests/__init__.py

```diff
@@ -0,0 +1,19 @@
+import configparser
+import os
+from pathlib import Path
+
+TESTING_PROFILE = 'jccli-dev-testing'
+CONFIG_FILE_PATH = str(Path.home().joinpath('.jccli.ini'))
```

Comment:
You could import this from config.py to avoid the duplication.

Reply:
are you planning to implement this @jkiang13 suggestion?

Reply:
Yep, implemented: https://github.com/Sage-Bionetworks/jccli/pull/70/files#diff-164cf6e39279302ce61673fe98356a3cae092022e7701023974cd0725026fece

Reply:
Yep, implemented it.